keyboards/davinci_v1/v1_standard_via_layout.json

```diff
@@ -0,0 +1,204 @@
+{
```

Comment:
Remove all of the VIA json files. They do not belong in the QMK repo, and are not supported nor used by it.